imports/plugins/included/shipping-rates/server/no-meteor/queries/getFlatRateFulfillmentRestriction.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name getFlatRateFulfillmentRestriction
+ * @method
+ * @memberof Fulfillment/NoMeteorQueries
+ * @summary Query the FlatRateFulfillmentRestrictions collection for restrictions with the provided shopId
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} params - request parameters
+ * @param {String} params.shopId - Shop ID for the shop that owns the restriction
+ * @return {Promise<Object>|undefined} - A restrictions document, if one is found
+ */
+export default async function getFlatRateFulfillmentRestriction(context, { restrictionId, shopId } = {}) {
+  const { collections } = context;
+  const { FlatRateFulfillmentRestrictions } = collections;
+
+  return FlatRateFulfillmentRestrictions.find({
+    _id: restrictionId,
+    shopId
+  });
```

Comment:
This will always return a single doc, so it should not return a cursor and the GraphQL should not be a connection. Just change `find` to `findOne` here.